JS/main.js

```diff
@@ -1 +1,49 @@
-'use strict'
\ No newline at end of file
+'use strict'
+
+const btn = document.getElementById('btn');
+const tasks = [];
+
+// 追加ボタン
+btn.addEventListener('click', () => {  
+  addTask();  
+});
+
+
+
+// ↓関数内容↓
+
+// タスクの追加
+const addTask = () => {
+  const task = {
+    id: tasks.length,
+    taskName: document.getElementById('taskForm').value,
+    status: '作業中',
+    remove: '削除',
+  }
+  tasks.push(task);
+
+  const tbody = document.querySelector('tbody');
+  const tr = document.createElement('tr');
+  tbody.appendChild(tr);
+  tr.className = 'working';
+  
+  const keys = Object.keys(task);
```

Comment:
`keys` という変数をこれ以降使わないので、以下のように記述しても良いのではと思いました。
```suggestion
  Object.keys(task).forEach(key => {
```

Reply:
こちらに関しては気づけていませんでした。アドバイスありがとうございます。^^

Reply:
修正内容 OK です！